gnosis-imap/gnosis-imap-spec.ini

```diff
@@ -494,6 +494,127 @@ gas: #gas(INITGAS, NONMEMGAS, MEMGAS) => #gas(INITGAS, NONMEMGAS +Int #sendGas(B
     andBool 0 <Int #REFUND_RECEIVER
     andBool #sendFailure(#REFUND_RECEIVER)
 
+
+
+[checkSignatures]
+pc: PC_FUN_BEGIN => PC_FUN_END
+wordStack: CONSUME_HASH : DATA_HASH : WS => RESULT : WS             ; some arguments on stack (?)
+localMem: MEM:IMap => storeRange(MEM, HAVOC_START, HAVOC_WIDTH, _)  ; localMem is updated but we don't care (?)
+proxy_storage: STORAGE:IMap                                         ; storage is not updated (?)
++requires:                                                          ; some arguments on memory (?)
+    andBool selectRange(MEM, DATA_LOC, DATA_WIDTH) ==K DATA_BUF
+    andBool selectRange(MEM, SIGS_LOC, SIGS_WIDTH) ==K SIGS_BUF
+    andBool select(STORAGE, #hashedLocation({COMPILER}, {THRESHOLD}, .IntList)) ==Int THRESHOLD
```

Comment:
since we give an upperbound to the length of the sig bufs, we may also want to give an upperbound to the threshold instead of pow256.